meeshkan/serve/mock/specs.py

```diff
@@ -16,27 +17,50 @@ class OpenAPISpecification:
     source: str
 
 
-def load_specs(specs_dir: str) -> Sequence[OpenAPISpecification]:
-    if not os.path.exists(specs_dir):
-        logging.info("OpenAPI schema directory not found %s", specs_dir)
-        return []
+def load_spec(spec_source: str, is_http: bool) -> OpenAPISpecification:
+    spec_text: str
+    if is_http:
+        try:
+            response = requests.get(spec_source)
```

Comment:
Maybe check if the URL points to a `yaml` or `json` file?

Reply:
I've worked with a (admittedly crappy, and don't know how common that is nowadays) system that removed all file extensions in urls. So I'm holding off on this check for now.